### PR TITLE
[Kubernetes][Dashboards] Fix missing sort field on proxy and scheduler dashboards

### PR DESCRIPTION
**Pull request by constanca-m**
## What does this PR do?

Fixes an incorrect visualization:
image

That is present in both scheduler and proxy dashboards.

It should work as expected now:
image

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

### CODE REVIEW

File: packages/kubernetes/changelog.yml [hunk 1]
```diff
@@ -9,6 +9,11 @@
     - description: Remove container.name as a dimension.
       type: enhancement
       link: https://github.com/elastic/integrations/pull/6209
+- version: "1.38.2"
```
[the remark this answers is not in the document]
Comment (constanca-m) [reply packages/kubernetes/changelog.yml:12]: Interesting, I did not know that. I will update to 1.39.1 then

[the remark this answers is not in the document]
Comment (tetianakravchenko) [reply packages/kubernetes/changelog.yml]: @jsoriano could you please double-check this? is my assumption correct regarding the package version rebuilds?

[the remark this answers is not in the document]
Comment (jsoriano) [reply packages/kubernetes/changelog.yml]: Any new version would be published on merge, so if none of them are published, then both would work in that regard.
Said that, I would go with the proposal of using `1.39.1`, so you can still reserve `1.38.X` for backports to this minor if needed at some point.

If you need this fix in 1.38 for some reason, I would suggest to merge it first as `1.39.1`, and after that follow this procedure to release a `1.38.2` version: https://github.com/elastic/integrations/blob/ce78cfe4a13c14528bb88b809d7ed2789dd1b2ab/docs/developer_workflow_bug_fix_older_package_version.md

File: packages/kubernetes/changelog.yml [hunk 2]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.40.1-beta"
```
Comment (jsoriano) [inline packages/kubernetes/changelog.yml:2]: I would suggest to use `beta.1`, 1.40.0 hasn't been released.
```suggestion
- version: "1.40.0-beta.1"
```

**Review by tetianakravchenko — APPROVED** [review]
Tested upgrade `1.39.0` to `1.40.0-beta.1` locally - corresponding visualizations do no have `Sort field  was not found.` errors

File: packages/kubernetes/changelog.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (tetianakravchenko) [reply packages/kubernetes/changelog.yml]: thank you for the clarifications on the process!

**Pull request merged**